core_lib/web_helpers/flask/require_login.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+import logging
+from flask import request
+
+from core_lib.session.security_handler import SecurityHandler
+from core_lib.web_helpers.decorators import handle_exception
+logger = logging.getLogger(__name__)
+
+
+class RequireLogin(object):
+    def __init__(self, policies: list = []):
+        self.policies = policies
+

```

Comment:
yes create 2 decorators 
But use this common function please

`require_login_helpers.py`
```
def require_login(request, func, policies, *args, **kwargs):
    response = handle_exception(SecurityHandler.get()._secure_entry, request, policies)
    if not response:
        try:
            return func(request, *args, **kwargs)
        except Exception:
            logger.error(
                f'error while loading target page for controller entry name `{func.__name__}`', exc_info=True
            )
    return response
```